schema.json

```
@@ -37,6 +37,10 @@
                 "disableManagedObjectAliases": {
                     "type": "boolean",
                     "description": "Disables adding an [alias](https://www.pulumi.com/docs/intro/concepts/resources/options/aliases/) resource option to managed objects in the bucket."
+                },
+                "includeHiddenFiles": {
+                    "type": "boolean",
+                    "description": "Include hidden files (\"dotfiles\") when synchronixing folders. False by default."
```

Just to align with the others:

```suggestion
                    "description": "Include hidden files (\"dotfiles\") when synchronizing folders. Defaults to `false`."
```

---

I can't even blame this on US/UK spelling differences 🤦